entc/gen/template/client.tmpl

```diff
@@ -79,6 +80,7 @@ func (c *Client) Tx(ctx context.Context) (*Tx, error) {
 	}
 	cfg := config{driver: tx, log: c.log, debug: c.debug, hooks: c.hooks}
 	return &Tx{
+		ctx: ctx,
```

Comment:
If we store a context in tx maybe we should add `Context() / WithContext()` so users can modify it